system-design/orchestration/workload/workload-management-interface-breakdown.md

```diff
@@ -0,0 +1,25 @@
+# Margo Management API
+The [Margo management API](../../margo-api-reference/margo-api-specification.md) is a critical component that enables interoperability between devices and workload orchestration solutions. This API MUST be used for communication between all Margo compliant devices and orchestration services. This documentation section focuses on the workload management functions of this management API. The device's management client implementation can either be pre-packaged by the device manufacturer during production or installed by the device integrator later. 
+
+## Workload Management API Requirements:
+
+- The [Margo management API](../../margo-api-reference/margo-api-specification.md) MUST be used for the following core functions
+	- device onboarding with the workload orchestration solution
+   	- device capabilities reporting
+	- identifying desired state changes
+	- deployment status reporting
+- The workload orchestration solution vendors MUST implement a web service following the [Margo Management API specification](../../margo-api-reference/margo-api-specification.md).
+- The device vendor MUST implement a client following the [Margo Management API specification](../../margo-api-reference/margo-api-specification.md).
+- The workload orchestration solution MUST maintain a Git repository to store the devices desired state.
+- The device's management client MUST retrieve the device's desired state from the device's assigned Git repository.
+- Both Web API and GitOps patterns MUST support extended device communication downtime. 
```

Comment:
Where are the GitOps patterns defined?

Reply:
This will be marked as an open item for now until the discussions around GitOps usage have been resolved.